project/app/static/webpack_sources/scss/pages/_hodan.scss

```diff
@@ -99,12 +99,96 @@
 }
 
 .citizen-filter {
+    &>div{
```

Comment:
This css is for spacing on mobile I guess
![image](https://user-images.githubusercontent.com/9761138/98019253-1981c600-1e34-11eb-8ac8-c38979cd13d3.png)
Can we change it by adding `mb-2` class to each column inside `.citizen-filter`?